### PR TITLE
Removing deprecation messages in TableViewCell

### PR DESCRIPTION
**Pull request by sophialee0416**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Removing deprecation messages and replacing with comments to prevent breaking downstream dependencies.

### Verification

Build + pod lib lint

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1026)

### CODE REVIEW

**Comment by huwilkes** [comment]
Why are the deprecation messages being turned in to comments?

File: ios/FluentUI/Table View/TableViewCell.swift [hunk 1]
```diff
@@ -1081,22 +1081,23 @@ open class TableViewCell: UITableViewCell {
         }
     }
 
-    @available(*, deprecated, message: "Any color or stylistic changes on TableViewCell labels should be done through NSAttributedString.")
+    // Any color or stylistic changes on TableViewCell labels should be done through NSAttributedString
```
Comment (huwilkes) [inline ios/FluentUI/Table View/TableViewCell.swift:1084]: Should we name the actual property instead of its type?

Comment (huwilkes) [inline ios/FluentUI/Table View/TableViewCell.swift:1084]: Should we just be taking the color they give here and just applying it to the correct attributed string?

Comment (sophialee0416) [reply ios/FluentUI/Table View/TableViewCell.swift:1084]: I don't think so, I think we want to deter clients from using this in the future so eventually, and hopefully, we'll be able to remove this?

**Review by rdeassis — APPROVED** [review]

**Comment by sophialee0416** [comment]
> Why are the deprecation messages being turned in to comments?

Resolved offline + updated description

**Pull request merged**